### PR TITLE
add cases of the tooltip visibility

### LINKED ISSUE #1816: [Profile] Do not show tooltip for Total trades when no trades
Related to #1721, #1575

Currently, I see the tooltip that I can see 'more trades..' when I have a dash instead of the value in the Total trades section. 

It would be nice not to show it when: 
 - a user is not connected 
 - a user is not connected to the Mainnet
 - totally new user without trades

### PR DESCRIPTION
**Pull request by maria-vslvn**
# Summary

Fixes #1816 

total trades tooltip is always visible, and it shouldnt be visible when:
-a user is not connected
-a user is not connected to the Mainnet
-totally new user without trades

  # To Test

1. Open the page `Profile`
2. follow the conditions below to see the tooltip works when user is connected, the network is mainnet, trades numbers is higher than 0

### CODE REVIEW

**Comment by maria-vslvn** [comment]
@elena-zh @alfetopito @alongoni please check

**Review by elena-zh — APPROVED** [review]
LGTM!

File: src/custom/pages/Profile/index.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ export default function Profile() {
   const { account, chainId } = useActiveWeb3React()
   const { profileData, isLoading, error } = useFetchProfile()
   const lastUpdated = useTimeAgo(profileData?.lastUpdated)
-
+  const isTTradesTooltipVisible = account && chainId == 1 && parseInt(formatInt(profileData?.totalTrades)) > 0
```
Comment (alfetopito) [inline src/custom/pages/Profile/index.tsx:33]: There's a typo in the variable name.
Also, you can simplify the last part with `!!profileData?.totalTrades`

**Comment by anxolin** [comment]
The base branch is wrong for this PR. Pls be careful, if we merge to the wrong branch is likely we will loose the PR since we are working on serveral versions at the same time

right now we work in 1.4.11 and 1.5.0 at the same time...

File: src/custom/pages/Profile/index.tsx (hunk 1 above)
Comment (maria-vslvn) [reply src/custom/pages/Profile/index.tsx]: done

**Review by alfetopito — APPROVED** [review]
Looks good!

Just remember to point the branch to `release/1.5` before merging

**Comment by maria-vslvn** [comment]
@elena-zh @anxolin @alfetopito please check

**Review by alongoni — APPROVED** [review]

**Review by alfetopito — APPROVED** [review]

**Pull request merged**